steam/ext/csgo/protobufs/ranks.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+
+import betterproto
+
+from ....protobufs.msg import GCProtobufMessage
+from ..enums import EMsg
+
+
+@dataclass(eq=False, repr=False)
```

Comment:
This decorator isn't necessary

Reply:
What about the second decorator?  Also not necessary? ` @dataclass(eq=False, repr=False)`

Reply:
No the second one is because it subclasses betterproto.Message directly

Reply:
Do we actually need a new method?
We already got this one:

```py
    async def fetch_user_csgo_profile(self, user_id: int) -> cstrike.PlayersProfile:
        await self.ws.send_gc_message(cstrike.ClientRequestPlayersProfile(account_id=user_id, request_level=32))
        return await self.ws.gc_wait_for(
            cstrike.PlayersProfile,
            check=lambda msg: msg.account_profiles[0].account_id == user_id,
        )
```